src/deployers/PlatformPluginsDeployer.cpp

```diff
@@ -17,9 +18,19 @@ bool PlatformPluginsDeployer::deploy() {
 
     ldLog() << "Deploying platform plugins" << std::endl;
 
+    // always deploy default platform
     if (!appDir.deployLibrary(qtPluginsPath / "platforms/libqxcb.so", appDir.path() / "usr/plugins/platforms/"))
         return false;
 
+    // deploy extra platform plugins, if any
+    if (const auto* const platformPluginsFromEnvData = getenv("EXTRA_PLATFORM_PLUGINS")) {
```

Comment:
I *hate* implicit comparisons, I really should introduce such a compiler flag. Could you please rewrite the code to check explicitly against `nullptr`?

Reply:
Done